CONTRIBUTING.md

```diff
@@ -17,9 +17,28 @@ contributing to `kubevela` or build a PoC (Proof of Concept).
 2. Kubernetes version v1.16+ with `~/.kube/config` configured.
 3. ginkgo 1.14.0+ (just for [E2E test](./CONTRIBUTING.md#e2e-test))
 4. golangci-lint 1.31.0+, it will install automatically if you run `make`, you can [install it manually](https://golangci-lint.run/usage/install/#local-installation) if the installation is too slow.
+5. kubebuilder v2.X.X
 
-We also recommend you to learn about KubeVela's [design](https://kubevela.io/docs/concepts) before dive into its code.
+##### kubebuilder install:
 
+linux:
+```
+wget https://github.com/kubernetes-sigs/kubebuilder/releases/download/v2.3.1/kubebuilder_2.3.1_linux_amd64.tar.gz
+tar -zxvf  kubebuilder_2.3.1_linux_amd64.tar.gz
+sudo mv kubebuilder_2.3.1_linux_amd64 /usr/local/kubebuilder
```

Comment:
```suggestion
sudo mv kubebuilder_2.3.1_linux_amd64/bin /usr/local/kubebuilder
```

Reply:
> 

i tested like this,  'make test' failed with "failed to start the controlplane. retried 5 times: fork/exec /usr/local/kubebuilder/bin/etcd: no such file or directory" .

Reply:
pls check it again. We need to move k8s control plane bins into `/usr/local/kubebuilder/bin`

Reply:
first, i run 'make test' success.

then, i test like this, and 'make test' failed again.
```
➜  kubevela cd /usr/local/kubebuilder
➜  kubebuilder cp -r bin/ ./
➜  kubebuilder ls
bin            etcd           kube-apiserver kubebuilder    kubectl
➜  kubebuilder ls bin
etcd           kube-apiserver kubebuilder    kubectl
➜  kubebuilder mv bin bin-back
➜  kubebuilder tree
.
├── bin-back
 │   ├── etcd
 │   ├── kube-apiserver
 │   ├── kubebuilder
 │   └── kubectl
├── etcd
├── kube-apiserver
├── kubebuilder
└── kubectl

 export PATH="/usr/local/kubebuilder:$PATH"
```

Reply:
your command will move the whole `kubebuilder_2.3.1_linux_amd64` dir, but we just need move `bin` dir, right?

Reply:
you mean create kubebuilder dir and mv  bin dir ? why not move and rename redirectly ?

Reply:
if users already have kubebuilder dir, this cmd will move dir into it instead of renaming.

Reply:
ok, thanks